Parsers/ASimAuditEvent/Parsers/ASimAuditEventCiscoISE.yaml

```diff
@@ -0,0 +1,295 @@
+Parser:
```

Comment:
can you please add entry for this parser in union parser?
https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimAuditEvent/Parsers/ASimAuditEvent.yaml
https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimAuditEvent/Parsers/imAuditEvent.yaml

Reply:
rest all looks good

Reply:
Added records.
P.S. I wonder if the records are correct in `imAuditEvent.yaml` unified parser (see screenshot below). Shouldn't they start with `vim`?
<img width="1061" alt="image" src="https://github.com/Azure/Azure-Sentinel/assets/94110440/35acb13f-a35b-411d-b294-8082e8c13610">

Reply:
I'll look into this, can you follow the existing format for now like other audit parsers?

Also, please add ',' at the end of AzureActivity parser

Reply:
Changed to existing format like for other parsers